### PR TITLE
Interactive 

### PR DESCRIPTION
**Pull request by Rikol137**
- Used Gitflow used.
- Linters used and checks are done.
- Interactive files added to the src file inside the modules file.
- Update done on the index.js and style.css to allow the interactive files to perform.

### CODE REVIEW

File: src/modules/newTask.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/extensions
```
Comment (emmiiorji) [inline src/modules/newTask.js]: Your project looks great. As much as possible, kindly avoid disabling linter checks in line. They represent coding standards you share with your team and for most of the time enforce adherence to best coding practices.

File: src/modules/removeTask.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/extensions
```
Comment (emmiiorji) [inline src/modules/removeTask.js]: As mentioned above, kindly avoid disabling linter checks inline. Remember to remove all other occurrences of it.

**Review by emmiiorji — APPROVED** [review]
Hi @Rikol137, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more

## Good Highlights✔

- Linter checks are passing
- Good commit messages
- List is interactive


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/displayTask.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    listContainer.className = 'allTodos';
+    listSection.appendChild(listContainer);
+    todo.allTodos.map((a) => {
+      const list = document.createElement('li');
+      list.className = 'todo';
+
+      const descrptContainer = document.createElement('div');
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.id = 'checkbox';
```
Comment (OLIPLICHE) [inline src/modules/displayTask.js:20]: Nice 👍 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @Rikol137, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- The App is working fine👏 👏
- Applied the required change👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @Rikol137, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**